net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardClient.xml

```diff
@@ -0,0 +1,40 @@
+<form>
+    <field>
+        <id>client.enabled</id>
```

Comment:
client -> peer

Reply:
Can you give me a proper expl. of all? First I was thinking of server/clients. Now I'm on Server and Endpoint .. makes it clearer for the enduser. How would you differentiate between peer and endpoint?